aviary/interface/methods_for_level2.py

```diff
@@ -2325,6 +2327,169 @@ def run_aviary_problem(self,
 
         self.problem_ran_successfully = not failed
 
+    def run_alternate_mission(self, json_filename='sizing_problem.json',
+                              payload_mass=None, mission_range=None,
+                              phase_info=None, optimizer=None, max_iter=50,
+                              verbosity=Verbosity.BRIEF):
+        """
+        This function runs a fallout mission based on a sizing mission output.
+
+        Parameters
+        ----------
+        json_filename : str
+            Name of the file that the sizing mission has been saved to.
+        mission_range : float, optional
+            Target range for the fallout mission.
+        payload_mass : float, optional
+            Mass of the payload for the mission.
+        phase_info : dict, optional
+            Dictionary containing the phases and their required parameters.
+        optimizer : str, optional
+            Optimizer to be used for solving the fallout mission.
+        max_iter : int, optional
+            Max number of iterations allowed for the optimizer.
+        verbosity : Verbosity or list, optional
+            If Verbosity.DEBUG, debug print options ['desvars','ln_cons','nl_cons','objs'] will be set.
+            If a list is provided, it will be used as the debug print options.
+        """
+        if phase_info is None:
+            phase_info = self.phase_info
+        if mission_range is None:
+            design_range = self.get_val(Mission.Design.RANGE)
+        if payload_mass is None:
+            if self.mission_method is HEIGHT_ENERGY:
+                payload_mass = self.get_val(Aircraft.CrewPayload.TOTAL_PAYLOAD_MASS)
+            elif self.mission_method is TWO_DEGREES_OF_FREEDOM:
+                payload_mass = self.get_val(Aircraft.CrewPayload.PASSENGER_PAYLOAD_MASS)
+        if optimizer is None:
+            optimizer = self.driver.options["optimizer"]
+
+        mission_mass = self.get_val(Mission.Design.GROSS_MASS)
+
+        prob_alternate = _load_off_design(json_filename, ProblemType.ALTERNATE,
+                                          phase_info, payload_mass, design_range, mission_mass)
+
+        prob_alternate.check_and_preprocess_inputs()
+        prob_alternate.add_pre_mission_systems()
+        prob_alternate.add_phases()
+        prob_alternate.add_post_mission_systems()
+        prob_alternate.link_phases()
+        prob_alternate.add_driver(optimizer, max_iter=max_iter, verbosity=verbosity)
+        prob_alternate.add_design_variables()
+        prob_alternate.add_objective()
+        prob_alternate.setup()
+        prob_alternate.set_initial_guesses()
+        prob_alternate.run_aviary_problem(record_filename='alternate_problem_history.db')
+        return prob_alternate
+
+    def run_fallout_mission(self, json_filename='sizing_problem.json',
```

Comment:
General api thoughts -- do we want a separate function for each offdesign mission type, or do we want a single function with an mission type attribute? Asking mostly with the consideration of future types (if there are any others).